lib/bezier.js

```diff
@@ -795,6 +795,12 @@
           np3 = this.get(e);
 
           arc = utils.getccenter(np1, np2, np3);
+          
+          //also save the t values
+          arc.t_s = s;
```

Comment:
I'm adding Bezier curve functionality to Maker.js and plan on taking a dependency on Bezier.js :) 
Maker.js has Boolean operations for shapes. I'm finding that casting Bezier curves into arcs is making it much easier to reason. I am able to detect a chain of arcs. Some of the arcs may have been clipped. I will be able to cast this chain back into a Bezier by looking at the t-values of the outermost arcs.